pinguess/PinGuess.java

```diff
@@ -3,15 +3,51 @@
     //Variable Declaration
     
     //Method Declaration
+  
+      public String leftPad(Int number, Int targetLength) {
+        Int output = number + '';
```

Comment:
// there is a primitive data type int and an Object Integer in Java.. one of these should be used here (probably Integer so you can use the .toString() method